### DISCUSSION TITLE
Local Off

### DISCUSSION THREAD

**Discussion by puffer3** [image-et-son/p600fw#155] [Ideas]
I love my prophet 600 GLIGLI so much.  I even installed a new techsmecs keyboard.  It plays so nice.  When Controlling the arpeggiator in ableton tho I wish I could turn Local Midi Off.  That seems like a somewhat useful function that would be easy to implement, right? Does anyone else feel this way?

**Comment by Jpro600k** [comment]
Hi puffer3

About local on, local off
Press  'FROM TAPE' + 0 (zero) or  FROM TAPE + 00 (zero Twice) 
You can select On or Off.

Please check
p600fw_User_Manual_beta_2022_.1.pdf
page 64
5.3 Settings parameter overview

**Reply by puffer3** [reply]
Thanks for your kind response. i must have the older firmware then! is there a way to flash the teensy via sysex to update it or do i have to take it out and plug in the usb?

**Reply by Jpro600k** [reply]
Back up your presets before updating.

It seems that some users can update it by simply sending firmware.syx via MIDI without Flash. Why not send the latest p600firmware_beta_2022_.1syx and give it a try?

Read this.
p600fw_User_Manual_beta_2022_.1.pdf
Page 58 
Firmware upgrade via SysEx 

To receive the preset, press FROM TAPE + RECORD, and then send the SYX of the preset from the PC with the Preset button blinking. On P600, When the count up from 00 to 99 is complete, press the RECORD button. The preset is now registered.

Read this.
p600fw_User_Manual_beta_2022_.1.pdf
Page 55,56 
Managing sound libraries via SysEx

If your CPU isn't working well, flashing is likely to help.

**Reply by puffer3** [reply]
My CPU is working great! I think I just must have used the older firmware.  Where can I find the .1syx file?

**Comment by matrix12x** [comment]
https://github.com/image-et-son/p600fw/releases/tag/v2022-beta.1

**Reply by GrantMTG** [reply]
Is it possible to copy or rewrite the update instructions right into the new manual? These steps are very important for success, but it requires two manuals to follow:

_You are recommended to follow the MIDI SysEx part of the modification instructions [4]. In particular, the ”Delay between buffers” and the ”delay after F7” should be set to at least 250ms._

**Reply by Jpro600k** [reply]
> Is it possible to copy or rewrite the update instructions right into the new manual? These steps are very important for success, but it requires two manuals to follow:
> 
> _You are recommended to follow the MIDI SysEx part of the modification instructions [4]. In particular, the ”Delay between buffers” and the ”delay after F7” should be set to at least 250ms._

Since the description is almost the same as reading, it seems that it is not necessary to read the two, but if you kindly add it, I think that it is easier to understand if you describe a concrete example.
You may be wondering if you should write the explanation so carefully that it is too kind. It may not be desirable to increase the description of other software other than the function.

If you want to add something, it may be convenient to describe the hierarchy of the menu.
like a this example ?

***
For　MIDI OX
Pulldown menu>View>Select SysEx>
Open SysEx view window>Select pulldown menu Sysex>Select Configure
***
The SysEx settings are not in the main window, but in the menu of the SysEx window in the lower hierarchy, so this is a part that can be confusing when used for the first time.
